### PR TITLE
Feature/terraform

### PR DESCRIPTION
**Pull request by ryan-wise**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -4,12 +4,21 @@ inputs:
...
     required: true
+  terraform_directory:  
+    description: Directory containing Terraform files to be applied
+    required: true
+  terraform_workspace:  
+    description: The Terraform workspace of the changes to be applied
+    required: true
+  terraform_var_file:  
+    description: Location of the terraform var file
+    required: true
```
Comment (constantincerdan) [inline action.yml:21]: Should these be optional for repos that don't have any terraform resources?

Comment (constantincerdan) [inline action.yml:21]: Should these be optional for repos without terraform resources?

Comment (constantincerdan) [inline action.yml:21]: Should these be optional for repos without terraform resources?

Comment (ryan-wise) [reply action.yml:21]: Yeah - actually everything is optional. Will update

**Review by constantincerdan — APPROVED** [review]

**Pull request merged**